### DISCUSSION TITLE
is there a way to transfer credentials from requirements.txt when creating pipfile for the first time?

### DISCUSSION THREAD

**Discussion by vishal180618** [pypa/pipenv#5422] [Q&A]
I created a new Pipfile using `pipenv install -r requirements.txt`,  and it was stuck forever. The reason was there was credentials present in the GitHub URL and the `Pipfile` was created with masked credentials hence it was taking forever to fetch/install dependency using the masked credentials URL.

Is there should be a command to just create Pipfile only and not install dependency?
Is there a way I can transfer credentials from requirements to Pipfile?

**Comment by kalebmckale** [comment]
To create `Pipfile` without installing dependencies, run
```sh
pipenv --python=<version>
```

For a safer way to inject credentials, see https://pipenv.pypa.io/en/latest/credentials/
